app/webpacker/css/admin_v3/shared/forms.scss

```diff
@@ -1,5 +1,13 @@
 $text-inputs: "input[type=text], input[type=password], input[type=email], input[type=url], input[type=tel]";
 
+#{$text-inputs},
+input[type="date"],
+input[type="datetime"],
+input[type="time"],
+input[type="number"] {
+  height: $btn-relaxed-height;
```

Comment:
Setting a height like this worries me, but I think I've mentioned that before. I'd much prefer to set font size and padding to reach a desired height, which I think is more robust (for example if we want to change the font size on a particular field, it will "just work" without further overrides)

But I know it's harder to get the heights just right. And it's unlikely that we'd need to override this, so let's try it 👍

Reply:
ok, I (think i) see your point. `$btn-relaxed-height` is widely used, and seems to be very useful, easy to work with.